app/controllers/users_controller.rb

```diff
@@ -280,7 +280,11 @@ def determine_follow_suggestions(current_user)
       attributes_to_select: INDEX_ATTRIBUTES_FOR_SERIALIZATION,
     ).suggest
 
-    recent_suggestions.presence || default_suggested_users
+    if SiteConfig.prefer_manual_suggested_users?
+      default_suggested_users
+    else
+      recent_suggestions.presence || default_suggested_users
```

Comment:
```suggestion
      recent_suggestions || default_suggested_users
```
Since `nil` is falsy in ruby, if `recent_suggestions` is `false`, ruby should return the value of `default_suggested_users` (so you shouldn't need `presence` I believe!)

Reply:
Thanks for your suggestion! `presence` was in the original code before my change so I kept it. I'm not sure but I suppose we use `presence` here because `recent_suggestions` could be an empty array `[]` which is truthy?

Reply:
Ah, that makes much more sense! I think it's correct to leave it in, in that case 👍